### PR TITLE
users_mq_ipf AFC-1938: MQ listener for updated connections between of…

### PR DESCRIPTION
**Pull request by SebastianGieser-Vertel**
…fices and officers

### CODE REVIEW

File: users_mq_ipf/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    'version': '12.0.0.1.0',
+    'category': '',
+    'description': """
+
+
+""",
+    'author': 'Vertel AB',
+    'license': 'AGPL-3',
+    'website': 'http://www.vertel.se',
+    'depends': ['base', 'hr_office', 'af_process_log'],
```
Comment (daerio) [inline users_mq_ipf/__manifest__.py]: where does af_process_log come from? It's not present in my codebase.

**Review by daerio — CHANGES_REQUESTED** [review]
See comment.

File: users_mq_ipf/__manifest__.py (hunk 1 above)
Comment (SebastianGieser-Vertel) [reply users_mq_ipf/__manifest__.py]: https://github.com/vertelab/odoo-af/tree/12.0-AFC-2169

**Review by daerio — APPROVED** [review]
OK to merge after 12.0-AFC-2169 has been merged

**Review by daerio — APPROVED** [review]
looks ok to merge without12.0-AFC-2169 now

**Pull request merged**